### PR TITLE
Feature/xcode11

### LINKED ISSUE #137: Xcode 11 support
Hello !

Xcode 11 changes the format of the Test bundle files.
I know that Xcode 11 is able to re-open a Test bundle and display the content.
Nevertheless having an HTML version of the Test bundle is very handy.

Is there a plan to support the new format with XCTestHTMLReport ?

### PR DESCRIPTION
**Pull request by felginep**
Fix #137 

* Use Swift Package Manager to manage dependencies
* Use XCResultKit to parse `.xcresult` files
* Clean all Xcode 10 related code

### CODE REVIEW

**Comment by kenji21** [comment]
Maybe an issue of ResultKit: 

```
~/XCTestHTMLReport/.build/debug/xchtmlreport -r TestResults
XCTestHTMLReport 2.0.0
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus
Error parsing ActionTestMetadata: Element Not Found: testStatus

Report successfully created at ./index.html
```

Got a 404 on Logs tab because of absolute path on the CI server

**Comment by kenji21** [comment]
not tested attached screenshots yet

**Comment by felginep** [comment]
Yes logs are caused by `XCResultKit`. I'll propose a PR to remove / quiet them.
Good point for absolute paths, I will improve this to work on CI server. I guess you'll have the same issue with screenshots paths.

File: xchtmlreport.rb [hunk 1]
```diff
@@ -1,12 +1,12 @@
 class Xchtmlreport < Formula
   desc "XCTestHTMLReport: Xcode-like HTML report for Unit and UI Tests"
   homepage "https://github.com/TitouanVanBelle/XCTestHTMLReport"
-  url "https://github.com/TitouanVanBelle/XCTestHTMLReport/archive/1.6.1.tar.gz"
+  url "https://github.com/TitouanVanBelle/XCTestHTMLReport/archive/2.0.0.tar.gz"
```
Comment (Lewion) [inline xchtmlreport.rb]: This url doesn't seem to work, there does not seem to be a `2.0.0` archive?

Comment (felginep) [reply xchtmlreport.rb]: Indeed, I can't publish the 2.0.0. The PR needs to be accepted first.

**Comment by kenji21** [comment]
😱Xcode 11 xctestplans... (e.g. run your tests with multiple language):

xctestplans-with-2-configurations

but "xchtmlreporting" it works (not showing configuration but tests "duplicated")

Ref: https://developer.apple.com/videos/play/wwdc2019/413/

**Comment by felginep** [comment]
I did not support xctestplans yet. For now my priority was to provide support for projects that used the deprecated version of XCTestHTMLReport.
I'll will give a look to tests plans soon.

**Comment by kenji21** [comment]
feel free to ask me to do some testing

**Comment by cmarchal** [comment]
Thanks for your update !
I find issues about showing text attachment. Am I the only one facing it ?
Thanks again !

**Comment by felginep** [comment]
What is your issue exactly ? Can you check the source code of the html to find the path of the logs ?

**Comment by jineshqa** [comment]
@felginep Thank you for doing this. If may suggest, you need to update .travis.yml

```
osx_image: 
  - xcode11.1
```

**Pull request merged**